src/Core/Framework/DependencyInjection/seo.xml

```diff
@@ -152,5 +152,13 @@
             <argument type="service" id="category.repository"/>
             <tag name="twig.extension"/>
         </service>
+
+        <service id="Shopware\Core\Content\Seo\SeoUrlTwigFactory"/>
+
+        <service id="seo_url.twig" class="Twig\Environment">
+            <factory service="Shopware\Core\Content\Seo\SeoUrlTwigFactory" method="createTwigEnvironment"/>
+            <argument type="service" id="slugify"/>
+            <argument type="tagged" tag="seo_url.twig.extension"/>
```

Comment:
Since everything else is vendor specific I would prepend shopware here as well. Just my 2 cents.

```suggestion
            <argument type="tagged" tag="shopware.seo_url.twig.extension"/>
```

Reply:
sounds good to me